common/domain/handler.go

```diff
@@ -269,6 +269,7 @@ func (d *handlerImpl) RegisterDomain(
 	}
 
 	if activeClusters != nil {
+		// TODO: Leave a default activeClusterName for active-active domains
```

Comment:
you might want to either put your name there or some other identifier for the project as this might be hard to grep for